problem7/src/test/java/test/TestList.java

```diff
@@ -0,0 +1,33 @@
+package test;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import problem7.CustomLinkedList;
+
+
+public class TestList {
+
+	@Test
+	public void testRemoveDuplicates() {
+		CustomLinkedList list = new CustomLinkedList(50);
```

Comment:
I think this is overhead for test to populate it such way. Just put manually 5-7 elements